dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "backstage-community-plugin-redhat-argocd",
+  "version": "1.7.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public"
+  },
+  "backstage": {
+    "role": "frontend-plugin",
+    "supported-versions": "1.29.2",
+    "pluginId": "redhat-argocd",
+    "pluginPackages": [
+      "backstage-community-plugin-redhat-argocd"
+    ]
+  },
+  "exports": {
+    ".": "./src/index.ts",
+    "./package.json": "./package.json"
+  },
+  "typesVersions": {
+    "*": {
+      "package.json": [
+        "package.json"
+      ]
+    }
+  },
+  "scripts": {
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "clean-dynamic-sources": "yarn clean && rm -Rf node_modules",
+    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @backstage-community/plugin-redhat-argocd --override-interop default --in-place"
+  },
+  "dependencies": {
+    "@backstage-community/plugin-redhat-argocd": "1.6.7"
```

Comment:
The latest now is 1.8.0, I will create a followup PR to upgrade it in main branch and then backport it to 1.3.x.